src/urls.ts

```diff
@@ -1,9 +1,12 @@
 const baseURL = "https://github.com/DefinitelyTyped/DefinitelyTyped";
+const readmeLink = (header: string) =>
+    `${baseURL}#user-content-${header.replace(/[^\w -]/g, "").replace(/ /g, "-").toLowerCase()}`;
```

Comment:
Alternatively we could link to e.g. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson where the section links that GitHub exposes work without the `user-content-` workaround?